src/styles/utils/function.scss

```
@@ -0,0 +1,28 @@
+@for $index from 1 through 31 {
+  .calendar-day:nth-child(#{$index})::before {
+    content: "#{$index}";
+  }
+}
+
+$weekdays:
+  "mon" 0,
+  "tue" 101px,
```

calculate this value
```suggestion
  "tue" $dayMargin,
```